lib/modules/manager/npm/index.ts

```diff
@@ -31,4 +31,6 @@ export const defaultConfig = {
   },
 };
 
+export const categories: Category[] = ['js'];
```

Comment:
Why wouldn't this one be `node` too? I'm wondering if we really need both `node` and `js` though

Reply:
We tried to keep a 1:1 relation from languages to categories so far. If we break this, it makes sense to add this.     
I followed here the defined languages. There is a `node` language tag, but `npm` has `js` defined as programming language.

Reply:
Seems wrong in hindsight though. What if we just use `js` and migrate `node` later?

Reply:
Sounds good to me.